gnosis/safe-react
# [L2-UX] Fix multiple clicks on switching network

## What it solves
Resolves #2724. PR #2734 has to be merged first.

## How this PR fixes it
Fixes onboard initialization. It also updates onboard to the latest version so it fixes the repeated polling when switching networks.
Add some improvements to recover connection used last time on each network

## How to test it
 - Try switching networks.
 - Check that if Metamask was used last time connections is automatically recovered
 - Check that connection with Ledger/Trezor is possible in those compatible networks
 - Clicking multiple times in switch network doesn't trigger a continuous error, just once and show onboard modal second time (this is good to allow the user switch wallet in case wallet is not able to change network)

## Screenshots
